examples/external/plot_jit.py

```diff
@@ -28,10 +26,10 @@
 G.add_edge("two", 3, type="extra special")
 
 # convert to JIT JSON
-jit_json = jit_data(G, indent=4)
+jit_json = nx.jit_data(G, indent=4)
```

Comment:
This isn't something to be tackled in this PR, but having the name `jit_data` available from the main `nx` namespace feels like it could be misleading, given that `jit` is commonly associated with just-in-time compilation. I think `nx.json_graph.jit_data` is less ambiguous. I would recommend using that here and maybe consider removing `jit_data` from the top-level namespace in another issue/PR.

Reply:
Would the name `JIT_data` be better than `jit_data`? 
Is Javascript Infovis Toolkit the name we want to use here? Maybe there is a better name for the toolkit that could help us change our API to be less confusing. Now's the time to deprecate, right? :}

Reply:
> Would the name JIT_data be better than jit_data?

I'm not sure... it's an unfortunate collision of acronyms. If anything, the all-lowercase name is probably better, as "just-in-time" seems to be more commonly abbreviated as JIT (according to internet search results).

My thinking was that `json_graph` plays a role in disambiguating the acronym, so if the module was reorganized so that `jit_data` had to be accessed from `nx.json_graph` rather than directly from `nx` that would help. Of course, there's nothing stopping someone from doing `from networkx.json_graph import jit_data`, so it's not fool-proof. I'm not familiar with the Javascript infovis toolkit, so I'm not sure whether there's a good candidate for a more descriptive name.

Reply:
It looks like ``thejit`` could be used.  For me the first google hit for ``thejit`` is https://philogb.github.io/jit/.  It is also used on stackoverflow:  https://stackoverflow.com/questions/tagged/thejit

And http://thejit.org redirects to https://philogb.github.io/jit/.

However, the last commit to https://github.com/philogb/jit was Sept. 2014.  The last release was July 1, 2013.  The newest question on was Feb. 2015.   The mailing list is getting questions, but it looks like no one is answering.

We did get a recent PR:  https://github.com/networkx/networkx/pull/3891

How popular is the Javascript Infovis Toolkit.  It looks to me like it is a dead project.  It looks like the only way to install it is to download the source (or clone the repo) and then type ``python make.py build``, but ``make.py`` is a Python 2 script.

It looks like @MridulS contributed ``jit_data`` and ``jit_graph`` in #1945 based on #881.  Do you know more about if this widely used still?